.pre-commit-config.yaml

```diff
@@ -25,7 +25,7 @@ repos:
       - id: trailing-whitespace
 
   - repo: https://github.com/PyCQA/isort
-    rev: 5.10.1
+    rev: 5.12.0
```

Comment:
Not sure if `isort` is the only dependency that requires an update. Would like to address that in another thread, though.

Reply:
Yeah, I only update `isort` since it was causing the lint check to fail

Reply:
Yup, I've just noticed that. Weird issue though :/

Reply:
@DiddiLeija I've started https://github.com/wntrblm/nox/pull/690 to update all the pre-commit hooks.

Perhaps the pre-commit.ci integration should be added to the repo so `pre-commit autoupdate` is run automatically.

Reply:
> Perhaps the pre-commit.ci integration should be added to the repo so `pre-commit autoupdate` is run automatically.

I think we've discussed that before, am I right @wntrblm/nox @henryiii ?

Reply:
I'm a big fan of pre-commit.ci.

Poetry-core 1.5.0 broke isort. So you can't build anything before isort 5.12.0 from source. A little ironic given Poetry's stance on SemVer...

I'd like to move over to using Ruff soon, I'll probably make a PR in the near future.

Reply:
Big fan of ruff, would love to see that in Nox!

Reply:
+1 on Ruff 😄

Reply:
I don't know a lot about Ruff, but seems like a nice approach! :smile: